Code/RDGeneral/LocaleSwitcher.cpp

```diff
@@ -33,7 +33,6 @@
 // LocaleSwitcher Dependencies
 #include <clocale>
 #ifndef _WIN32
```

Comment:
BTW, aren't <clocale> and <locale.h> the very same header? in that case, we could probably also remove the #else part